commands/info.md

```diff
@@ -28,231 +28,231 @@ When no parameter is provided, the `default` option is assumed.
 
 ```valkey-cli
 127.0.0.1:6379> INFO
-# Server

```

Comment:
i tried editing via Github Web and it is still produced these changes. This seems to be a different save format (maybe it is the newline).

Reply:
GitHub is owned by Microsoft so maybe they want to force DOS/Windows newline to everyone? :dizzy_face: